jobs/harbor/templates/bin/pre-start.erb

```diff
@@ -166,5 +166,13 @@ fi
 #Save installed harbor version to file
 echo $HARBOR_FULL_VERSION > $HARBOR_VERSION_FILE
 
+<%- if p("enable_externalssl") -%>
+################# copy nginx.conf file
```

Comment:
By convention, we use one '#' to start a comment. This code block is simple enough to not having a block comment like "#################  ... ...  ################# "